LSNBLDR-863 Fix so embedded announcements follow the set colour scheme.

RebeccaOxfordUni opened the pull request:
Pre-fix:
![prefixred](https://user-images.githubusercontent.com/17832659/30702404-bc912066-9ee4-11e7-9c3d-ed08e22c7f12.JPG)
Post-fix:
![postfixredjpg](https://user-images.githubusercontent.com/17832659/30702411-c2c932de-9ee4-11e7-8a1a-de81c55d5411.JPG)
![postfixgrey](https://user-images.githubusercontent.com/17832659/30702417-c5c74f34-9ee4-11e7-868c-4b4cd8aaaa21.JPG)

buckett commented on lessonbuilder/tool/src/webapp/css/Simplepagetool.css:
```diff
@@ -522,7 +522,7 @@ a.edit-link:link, a.edit-link:visited, a.section-merge-link:link, a.section-merg
    color: #4b2600;
 }
-.colyellow .commentDiv, .colyellow .studentContentTable, .colyellow .questionDiv .contentCol, .colyellow .checklistDiv. colyellow .forumSummaryHeaderDiv, .colyellow .announcementsHeaderDiv .timeline-Header{
+.colyellow .commentDiv, .colyellow .studentContentTable, .colyellow .questionDiv .contentCol, .colyellow .checklistDiv. colyellow .forumSummaryHeaderDiv, .colyellow .announcementsHeaderDiv, .timeline-Header{
```
I think this is also wrong:
`.colyellow .checklistDiv. colyellow .forumSummaryHeaderDiv` 
while we're here shall we just fix this as well.

RebeccaOxfordUni commented:
Fixed the colyellow and colgreen selectors as they are obviously not right.
Have also removed the class timeline-Header as it is not used in the HTML.
